README.md

```diff
@@ -1,141 +1,146 @@
+
+To avoid this organization's need a mechanism to impose a constraint that can be enforced by the actions of any individual within the organization rather than a majority of participants.
+
+### Proposal Agreements
+
+Proposal Agreements are designed to facilitate these types of constraints within Aragon organizations. They allow an organization to define human readable terms and require proposers to deposit collateral before the users proposal can be forwarded to a voting app.
+
+The human readable terms can be used to protect the interest of minority stakeholders as described in the previous section, but they can also be used to define basic quality standards for what supplemental information must be included with a proposal.
+
+Proposal agreements can be paired with a Voting app by assigning the *create vote*, *pause vote*, and *cancel vote* roles.
+
+![](images/proposal-agreement-permissions.png)
+
+*Figure 2: Proposal Agreement --> Voting --> Finance --> Vault*
+
+With this flow, when a user wants to make a fund transfer they will be prompted to review and agree to the terms of the proposal agreement.
+
+### Dispute Creation
+
+If a minority stakeholder feels that a submitted proposal has violated the proposal agreement terms they can choose to raise a dispute. When they raise a dispute they will need to deposit an equivalent amount of collateral, along with initial *dispute fees* determined by the Aragon Court. They can also provide *evidence* to support their position. The vote will be immediately paused until the dispute is resolved.
+
+If the original proposer feels that the dispute is valid they can opt to do nothing and the dispute will be automatically ruled in favor of the disputer. The proposers collateral will be transferred to the disputer and the vote will be canceled. If the original proposer believes they will win the dispute then they must also deposit *dispute fees* and provide evidence to support their position.
+
+Once both parties have submitted evidence and dispute fees, the case is scheduled to be reviewed.
+
+## Aragon Court Protocol
+
+Proposal Agreement disputes rely on a decentralized oracle protocol referred to as the *court* where *jurors* stake the Aragon Network's native asset ANT in order to earn the right to perform dispute resolution service and earn a portion of *dispute resolution fees*.
+
+When a dispute occurs, a jury is formed by drafting jurors via *stake-weighted sortition*. Drafted jurors are required to commit to a ruling on the dispute within a commitment period, and then reveal their ruling after all drafted jurors have committed. The verdict is returned based on the majority decision of drafted jurors.
+
+Before the verdict is enforced, there is an opportunity to *appeal*, which repeats the adjudication process with a larger set of jurors. Appeals can be made a fixed number of times before a *final judgement round* that requires every staked juror to commit to a ruling.
```

Comment:
```suggestion
Before the verdict is enforced, there is an opportunity to *appeal*, which repeats the adjudication process with a larger set of jurors. Appeals can be made a fixed number of times before a *final judgment round* that requires every staked juror to commit to a ruling.
```

Reply:
As we are using American spelling everywhere else, "judgment" would be the correct spelling here.